HLTrigger/Configuration/python/Tools/confdb.py

```diff
@@ -87,8 +87,8 @@ def __init__(self, configuration):
 
     self.labels = {}
     if self.config.fragment:
```

Comment:
`confdb.py` can use a major cleanup if these changes are accepted - for the moment I've limited myself to the minimal changes